### DISCUSSION TITLE
How To add QRZ.ORG API key

### DISCUSSION THREAD

**Discussion by se7en9057** [magicbug/Cloudlog#3202] [Q&A] [answered]
I'm new to CloudLog (hosted) and i'm looking for the field to add the QRZ API key. ive looked online and found that i need to add it in the account settings > 3rd party services. ive done both un/pw as well as un/APIkey with no results. any help would be appreciated.

test is done by trying to pull from QRZ.org

**Comment by magicbug** [comment, marked as answer]
After adding call book login details log out and then back in

**Comment by magicbug** [comment]
If you want to use the feature to upload to qrz you'll need to add API key to the Station Location
